airflow/providers/qubole/operators/qubole_check.py

```diff
@@ -27,21 +27,26 @@
 class _QuboleCheckOperatorMixin:
     """This is a Mixin for Qubole related check operators"""
 
+    kwargs: dict
+    results_parser_callable: Optional[Callable]
+
     def execute(self, context=None) -> None:
         """Execute a check operation against Qubole"""
         try:
             self._hook_context = context
-            super().execute(context=context)
+            super().execute(context=context)  # type: ignore[misc]
```

Comment:
I found a `better`? way of handling it. I think it's a bit nicer:

```
def execute(self: BaseOperator, context=None) -> None:  # type: ignore[misc]
```
It's not as spectacular as in my case where I got rid of 5 self.* references this way, but I think it is more explicit (the ignore above has to be added silence another MyPy error "BaseOperator is not a superclass of Mixin'

https://github.com/apache/airflow/pull/20329/files#r770023071

Reply:
Nice solution :)
It is very similar to using Protocol but without protocol. Fortunately, it also works for my case. 
I think might be cases where some class inherits from mixin and two other classes and these classes have such different attributes and can't be reduced to one. But it will be another story.
Thanks for the suggestion

Reply:
Nice solution :)
It is very similar to using Protocol but without protocol. 
Unfortunately, it doesn't work for me.
In this case, `BaseOperator` hasn't method `get_hook` as well as none of the `super` classes have `execute` method. 

I think might be cases where some class inherits from mixin and two other classes and these classes have such different attributes and can't be reduced to one. But it will be another story.
Thanks for the suggestion